docs/src/development/sanitize-db.md

```diff
@@ -0,0 +1,143 @@
+
+Databases of live websites often contain Personally Identifiable Information (PII) such as full name, mailing addresses, ... .
+To avoid any unauthorized access to PII on the development environment, you can sanitize the database.
+
+## Before you begin
+
+This guide is about sanitizing databases.
+This guide doesn't address neither input validation nor input sanitization, an introduction to these topics can be found in the (free) [Open Web Application Security Project (OWASP) cheat sheet](https://cheatsheetseries.owasp.org/cheatsheets/Input_Validation_Cheat_Sheet.html#goals-of-input-validation).
+The `log4j` security vulnerability is an example of what missing input sanitization can cause.
+
+NoSQL Databases ([MongoDB](../configuration/services/mongodb/_index.md), ...) can benefit from sanitization but for the sake of simplicity, 
+these use cases won't be covered in this guide.
+
+You'll need:
+
+- a project that uses a [MySQL database](../configuration/services/mysql/_index.md),
+- the [Platform CLI](/development/cli/_index.md#cli-command-line-interface) installed locally.
+
+## Sanitize the Database
+
+Make sure that the sanitization only runs on the development environments and **never** on the production environment.
+Otherwise you will loose most — if not all — of the relevant data stored in your database.
+
+{{< codetabs >}}
+
+---
+title=Manually
+file=none
+highlight=false
+---
+
+Assumptions:
+
+- `staging` is the name of the database that you want to sanitize.
+- The `staging` database is an exact copy of your production database.
+- `user` is the table where all your PII is stored.
+
+1. As a safety precaution, take a [database dump](../configuration/services/mysql.md#exporting-data). Run `platform db:dump -e staging` to get a database dump of your staging environment,
+1. Connect to the `staging` database with `platform sql -e staging`,
+1. Display all fields from your `user` database with `SELECT * FROM users;`.
+   Which could result in something along the lines of:
+
+   ```sql
+   MariaDB [main]> SELECT * FROM users;
+   +----+------------+---------------+---------------------------+---------------+
+   | ID | first_name | last_name     | user_email                | display_name  |
+   +----+------------+---------------+---------------------------+---------------+
+   |  1 | admin      | admin         | admin@yourcompany.com     | admin         |
+   |  2 | john       | doe           | john.doe@gmail.com        | john          |
+   |  3 | jane       | doe           | janedoe@ymail.com         | jane          |
+   +----+------------+---------------+---------------------------+---------------+
+   3 rows in set (0.00 sec)
+   ```
+
+1. Change the fields where PII Data is contained with [`UPDATE` Queries](https://dev.mysql.com/doc/refman/8.0/en/update.html).
+   For example, changing the first name of an user, when the email address is not using the company domain, would result in the following query:
```

Comment:
```suggestion
   For example, to change the first name of a user with an email address not in your company's domain, run the following query:
```
The query doesn't result from the goal, it is a way to achieve it.
Also, is this for a single user or for any user where the `WHERE` condition is true? If the latter, the part about should be about `users` not `a user`.